toonz/sources/xdg-data/io.github.Tahoma.appdata.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.Tahoma</id>
+  <launchable type="desktop-id">io.github.Tahoma.desktop</launchable>
+  <name>Tahoma</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>BSD-3-Clause</project_license>
+  <developer_name>DWANGO Co., Ltd.</developer_name>
+  <summary>2D animation</summary>
+  <description>
+    <p>
+      This is software for producing a 2D animation.
+    </p>
+    <p>
+      It is based on the software "Toonz", which was developed
```

Comment:
Tahoma is based on OpenToonz which itself is based on Toonz, developed by Digital Video.

Reply:
Should I replace the entire text with this?

`Tahoma is based on OpenToonz which itself is based on "Toonz", developed by Digital Video S.p.A. in Italy and customized by Studio Ghibli.`

or just the line you gave me?

Reply:
I’m fine with either. Thanks.